tests/v3_api/resource/freeipa.json

```diff
@@ -52,5 +52,11 @@
   ],
   "testautogroupnested1": [
     "testauto1"
+  ],
+  "specialcharacter_users": [
+    "testauto20$",
+    "testauto21",
+    "testauto22",
+    "testauto25"
```

Comment:
Do the users testauto21/testauto22/testauto25 have  special characters  in some other properties? Otherwise we need not loop over them

Reply:
yes, testauto21/22 have special characters in first name/last name, testauto25 is normal, so I will remove testauto25